services/ui-src/src/shared/commonQuestions/DefinitionsOfPopulation.tsx

```diff
@@ -305,28 +307,30 @@ const CoreSetSpecificDefinitions = (
   register: any,
   labels: AnyObject,
   coreSetType: string,
-  coreSetOptions?: CoreSetSpecificOptions
+  coreSetOptions?: CoreSetSpecificOptions,
+  removeOtherOption?: boolean
 ) => {
   const options = coreSetOptions ?? coreSetSpecificOptions;
+  const otherOptions = [];
+  if (!removeOtherOption) {
+    otherOptions.push({
+      displayValue: "Other",
+      value: DC.DENOMINATOR_INC_OTHER,
+      children: [
+        <QMR.TextArea
+          formLabelProps={{ fontWeight: "400" }}
+          label={parseLabelToHTML(labels.defineDenomOther)}
+          {...register(DC.DEFINITION_DENOMINATOR_OTHER)}
+        />,
+      ],
+    });
+  }
```

Comment:
Thought: I usually like to initialize variables directly into their final state, whenever it's possible & reasonable. In this case, it's not too much trouble to initialize the `otherOptions` as having either 0 or 1 elements - versus initializing with 0 and maybe adding 1.

Writing it this way has the added benefit of removing the negation from the condition - `if`s and other conditionals are always a little easier for me to read when they don't include a `!`.

How do you like this?

```suggestion
  const otherOption = {
    displayValue: "Other",
    value: DC.DENOMINATOR_INC_OTHER,
    children: [
      <QMR.TextArea
        formLabelProps={{ fontWeight: "400" }}
        label={parseLabelToHTML(labels.defineDenomOther)}
        {...register(DC.DEFINITION_DENOMINATOR_OTHER)}
      />,
    ],
  };
  const otherOptions = removeOtherOption ? [] : [otherOption];
```

Reply:
Ooh I knew there existed a cleaner way to write that 😄